### DISCUSSION TITLE
Using `useField` outside of ValidatedForm

### DISCUSSION THREAD

**Discussion by wyemun** [airjp73/rvf#153] [Q&A]
In the docs at [https://www.remix-validated-form.io/integrate-your-components](https://www.remix-validated-form.io/integrate-your-components):
```
useField consumes the context provided by ValidatedForm, so these features only work if you use it in a ValidatedForm. However, it is still safe to use your input outside of a ValidatedForm, it just won't be able to take advantage of the features.
```

However, when I try to do this in actual app, it throws fatal error, contradicting the statement (`it is still safe to use`) above.

**Comment by airjp73** [comment]
Looks like the docs are outdated in this spot. Previously, I had implemented it so you could use your integrated components inside a plain remix `Form`, but I realized that this use-case wasn't worth the confusion this behavior caused. 

`useField` can still technically be used outside the context of a form, but it still needs to be associated with a form. See [using hooks outside forms](https://www.remix-validated-form.io/use-outside-forms).
